util/src/main/java/org/killbill/billing/util/audit/DefaultAccountAuditLogs.java

```diff
@@ -135,6 +135,11 @@ public AccountAuditLogsForObjectType getAuditLogs(final ObjectType objectType) {
         return auditLogsCache.get(objectType);
     }
 
+    @Override
+    public List<AuditLog> getAuditLogs() {
+        return ImmutableList.<AuditLog>copyOf(accountAuditLogs);
```

Comment:
It seems like we could change for free the definition of `accountAuditLogs` to be  `private final List<AuditLog> accountAuditLogs;` and avoid that copy.